sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/client/DefaultHttpClientResponse.java

```diff
@@ -37,10 +35,12 @@ public Headers getHeaders() {
     }
 
     @Override
-    public BinaryData getBody() {
-        return this.body;
-    }
     public HttpResponse buffer() {
         return this;
     }
+
+    @Override
+    public boolean isBuffered() {
+        return true;
```

Comment:
BinaryData does not have to be buffered
```suggestion
        return body.IsReplayable();
```